test/models/whois/record_test.rb

```diff
@@ -70,6 +70,6 @@ def test_updates_whois_record_from_auction_when_payment_received
   end
 
   def registration_deadline
-    Time.zone.now + 10.days
+    @registration_deadline ||= Time.zone.now + 10.days
```

Comment:
Now there is the tricky part.
Unless we cache the value we might face the situation when this method called in sloooow Travis VM and there is no guarantee two method calls will be made in the very same second.

So - we'll cache the value at first usage and when comparing will use cached value instead of new call for Time.zone.now.